### PR TITLE
sim_vehicle: add --callgrind option

### PR DESCRIPTION
**Pull request by peterbarker**

### CODE REVIEW

File: Tools/autotest/sim_vehicle.py [hunk 1]
```diff
@@ -654,6 +658,9 @@ def generate_frame_help():
     if cmd_opts.valgrind:
         print("May not use valgrind with hil")
         sys.exit(1)
+    if cmd_opts.callgrind:
+        print("May not use valgrind with callgrind")
```
Comment (OXINARF) [inline Tools/autotest/sim_vehicle.py]: Wrong message

Comment (peterbarker) [reply Tools/autotest/sim_vehicle.py]: Fixed

**Comment by peterbarker** [comment]
@OXINARF Fixed the sanity check on this one.

**Comment by OXINARF** [comment]
Merged, thanks!

**Pull request merged**